spec/02-integration/07-cache/database_cache_spec.lua

```diff
@@ -0,0 +1,84 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+local meta = require "kong.meta"
+
+describe("Resolver", function()
+  local client
+  setup(function()
+    helpers.kill_all()
```

Comment:
No need to call `kill_all()` when setting up tests anymore. Tests are responsible for doing this in their `teardown` phase.